### DISCUSSION TITLE
Welcome to ScreenTask Discussions!

### DISCUSSION THREAD

**Discussion by EslaMx7** [EslaMx7/ScreenTask#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by isma373** [comment]
Hey, I just tried on Windows 10 and it looks like not sharing Display 2. Is it just me? 

Also, quick question, do you think it would be a good idea to also implement Virtual Desktops (windows + Tab)  or even possible to do so? As potentially you could stream multiple virtual desktops by opening the application multiple times and choosing different ports.

**Comment by tc954** [comment]
Hey is there a way to configure the stream to default to full screen?

**Comment by DeconsterGod** [comment]
Hey guys is there any way to get connected without internet ? this software give bluetooth option and i tried it but i can't able to connect it and i also did't find any info about how to connect using bluetooth if anyone know how to do this please guied me i want to connect my pc screen to my phone but i don't have internet data.
